src/manifest/manifest_test.ts

```diff
@@ -32,6 +32,7 @@ Deno.test("SlackManifestType correctly resolves to a Hosted App when runOnSlack
     icon: "icon.png",
     botScopes: ["channels:history", "chat:write", "commands"],
   };
+
```

Comment:
All changes in this file seem unnecessary, my best guess is that they're remnants of when we didn't support `required` being `undefined` (w/ some formatting changes)

Reply:
Fixed in d09eb32